app.py

```diff
@@ -46,56 +45,71 @@ def main():
         
     if choice == "Home":
```

Comment:
By default Home should be the first thing we visit when the app loads as well as in the Menu Bar on the left

Reply:
Then what change should be done?

Reply:
The first if statement should be for Home Page

Reply:
okay.